Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -77,11 +129,31 @@ public void Activate(GameObject interactor)
 		{
 			this.enabled = true;
 			this.isMinimized = false;
+			this.isOptionsGuiOpen = false;
 			this.DeactivateAllParents ();
 			this.StartInteractions (interactor);
 		}
 	}
 
+	/// <summary>
+	/// Find the FirstPersonInteractor in the world, and use it to activate
+	/// 	the TwineNode's child at the given index.
+	/// </summary>
+	/// <param name="index">Index of the child to activate.</param>
+	private void ActivateChildAtIndex(int index) 
+	{
+		// Find the interactor:
+		FirstPersonInteractor interactor = (FirstPersonInteractor) FindObjectOfType(typeof(FirstPersonInteractor));
+
+		if (interactor != null) {
+			GameObject interactorObject = interactor.gameObject;
+
+			// Now activate the child using this interactor!
+			TwineNode child = this.children [index].GetComponent<TwineNode> ();
+			child.Activate (interactorObject);
```

Comment:
You _might_ be able to get away with passing null here. But I like how we're being safe!